src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
@MarshallOfSound Doesnt GPM uses a bigger album art for that fullscreen album cover thing?
[Heres an example](https://lh3.googleusercontent.com/1CAgnjsHDEsBXCryizRO5b3Am6xwMlzEk3bED_HdMfk8HyF598nG5SyZlUHpjD-owpEJPM0nBw=s1280-c-e100-nu)